### PR TITLE
No simple auth

### PR DESCRIPTION
**Pull request by sbrunner**
It's incoherent to have tow factor authentication and steel have the simple auth enable.
So we should be able to disable the simple auth, and by the way set it to disable by default.

### CODE REVIEW

File: doc/integrator/authentication.rst [hunk 1]
```diff
@@ -27,6 +27,19 @@ table. If a c2cgeoportal application should work with another user information
 source, like LDAP, a custom *client validation* mechanism can be set up.
 Our knowledge base has an example of how this can be achieved.
 
+Basic auth
+~~~~~~~~~~
+
+To be able acces to the OGC services from your desktop GIS you should anable the basic authentication
```
Comment (jwkaltz) [inline doc/integrator/authentication.rst]: To be able to access the OGC services from your desktop GIS, you should enable the basic authentication

File: doc/integrator/authentication.rst [hunk 2]
```diff
@@ -27,6 +27,19 @@ table. If a c2cgeoportal application should work with another user information
...
+by setting ``BASICAUTH`` to ``True`` in the ``.env.sample`` file.
+
+To force the application to ask for a password you should have the attribute ``authentication_required``
```
Comment (jwkaltz) [inline doc/integrator/authentication.rst]: To force the application to ask for a password, ...

File: doc/integrator/authentication.rst [hunk 3]
```diff
@@ -27,6 +27,19 @@ table. If a c2cgeoportal application should work with another user information
...
+in your querry string.
```
Comment (jwkaltz) [inline doc/integrator/authentication.rst]: query

File: doc/integrator/authentication.rst [hunk 4]
```diff
@@ -27,6 +27,19 @@ table. If a c2cgeoportal application should work with another user information
...
+
+.. note::
+
+   For security reason the basic auth and the two factors authentication shouldn't be enahabled toogether.
```
Comment (jwkaltz) [inline doc/integrator/authentication.rst]: For security reasons, basic authentication and two factor authentication should not be enabled together.

**Comment by sbrunner** [comment]
Thanks @jwkaltz

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 5]
```diff
@@ -7,7 +7,7 @@ Version 2.5.0
 Information
 -----------
 
-1.
+1. From now the basic auth is disabled by default.
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: To enable basic auth see: https://camptocamp.github.io/c2cgeoportal/master/integrator/security.html#basic-auth

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: +1 :-)

**Pull request merged**